numba/cuda/tests/cudadrv/test_nvvm_driver.py

```diff
@@ -25,6 +25,22 @@ def test_nvvm_compile_simple(self):
         self.assertTrue('simple' in ptx)
         self.assertTrue('ave' in ptx)
 
+    def test_nvvm_compile_nullary_option(self):
+        # Tests compilation with an option that doesn't take an argument
+        # ("-gen-lto") - all other NVVM options are of the form
+        # "-<name>=<value>"
+
+        # -gen-lto is not available prior to CUDA 11.5
+        if runtime.get_version() < (11, 5):
+            self.skipTest("-gen-lto unavailable in this toolkit version")
+
+        nvvmir = self.get_nvvmir()
+        ltoir = nvvm.llvm_to_ptx(nvvmir, opt=3, gen_lto=None, arch="compute_52")
+
+        # Verify we correctly passed the option by checking if we got LTOIR
+        # from NVVM (by looking for the expected magic number for LTOIR)
+        self.assertEqual(ltoir[:4], b'\xed\x43\x4e\x7f')
```

Comment:
It would be good to get a reference for this sequence but it's understandable if it's not possible.

Reply:
I agree - there is no reference, unfortunately.